src/kernels/dynamic_igemm/wrw_reduction_hip.cpp

```diff
@@ -0,0 +1,52 @@
+/*******************************************************************************
```

Comment:
instead of writing this new kernel,  did you evaluate using composable generic reduction added by Qianfeng?

https://github.com/ROCmSoftwarePlatform/MIOpen/pull/322

Reply:
I did not used generic reduction because it was under development when I opened this PR.
I am evaluating this kernel by using generic reduction API.

Reply:
@TejashShah The composable generic reduction also use tensordescriptor as template. The tensordescriptor contains the shape information of tensor, which means that we need to compile the kernel according to tensor length. So I don't think that it is a good solution for our dynamic igemm.

Reply:
Thanks for the explanation. It makes sense not to use generic reduction for our dynamic igemm.